### PR TITLE
Update multitenancy.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- updated the documentation based on https://github.com/dgraph-io/dgraph/pull/7789

### CODE REVIEW

File: content/enterprise-features/multitenancy.md [hunk 1]
```diff
@@ -444,16 +452,26 @@ mutation {
 }
 ```
 
-## Drop operations
+## Drop Operations
 
-The `drop all` and `drop data` operations can only be triggered by a [Guardian of the Galaxy](#guardians-of-the-galaxy).
+The `drop all` and `drop data` operations can be triggered by a [Guardian of the Galaxy](#guardians-of-the-galaxy).
```
Comment (NamanJain8) [inline content/enterprise-features/multitenancy.md:457]: Drop data operation is now namespace-aware. So the guardian of a namespace can trigger drop data for their namespace.

File: content/enterprise-features/multitenancy.md [hunk 2]
```diff
@@ -444,16 +452,26 @@ mutation {
...
 They're executed at cluster level and delete data across namespaces.
-All other `drop` operations run at namespace level and are namespace specific.
+All other `drop` operations run at namespace level and are namespace specific. For information about other delete options, see [delete](https://dgraph.io/docs/mutations/delete/). 
 
-{{% notice "note" %}}
-`drop all` and `drop data` operations are executed at cluster level and will delete across namespaces. The `drop data` operation will delete all the data but will keep the schema only.
+
+{{% notice "note:" %}}
+`drop all` and `drop data` operations are executed at cluster level and deletes across namespaces. Guardian of the namespace can trigger `drop data` operation within the namespace. The `drop data` operation deletes all the data but retains the schema only. Guardian of the namespace cannot trigger `drop schema` or `drop types` within the namespace.
```
Comment (NamanJain8) [inline content/enterprise-features/multitenancy.md:463]: Same here. Drop data is now namespace-aware. Here is the relevant PR https://github.com/dgraph-io/dgraph/pull/7795

File: content/enterprise-features/multitenancy.md [hunk 3]
```diff
@@ -444,16 +452,26 @@ mutation {
...
 {{% /notice %}}
 
+For example:
+
+```
+curl 'http://localhost:8080/alter' \
+  --data-raw '{"drop_op":"DATA"}' \
+  --compressed
+
+```
```
Comment (NamanJain8) [inline content/enterprise-features/multitenancy.md:473]: The examples are already [here](https://dgraph.io/docs/clients/raw-http/#alter-the-database). Do you think that adding a link should be sufficient here?

**Review by NamanJain8 — CHANGES_REQUESTED** [review]

File: content/enterprise-features/multitenancy.md (hunk 1 above)
Comment (Rajakavitha1) [reply content/enterprise-features/multitenancy.md:457]: This was only a word edit for Gaurdian of galaxy.

File: content/enterprise-features/multitenancy.md [hunk 4]
```diff
@@ -444,16 +452,27 @@ mutation {
...
 ```
 
-## Drop operations
+## Drop Operations
 
-The `drop all` and `drop data` operations can only be triggered by a [Guardian of the Galaxy](#guardians-of-the-galaxy).
+The `drop all` operations can be triggered only by a [Guardian of the Galaxy](#guardians-of-the-galaxy).
 They're executed at cluster level and delete data across namespaces.
-All other `drop` operations run at namespace level and are namespace specific.
+All other `drop` operations run at namespace level and are namespace specific. For information about other delete options, see [Alter the database]({{< relref "raw-http.md#alter-the-database" >}}).
```
Comment (NamanJain8) [inline content/enterprise-features/multitenancy.md:459]:
```suggestion
All other `drop` operations run at namespace level and are namespace specific. For information about other drop options, see [Alter the database]({{< relref "raw-http.md#alter-the-database" >}}). 
```

**Review by NamanJain8 — APPROVED** [review]
This looks good. Just make sure that the relative link works. 🎉

File: content/enterprise-features/multitenancy.md [hunk 5]
```diff
@@ -444,16 +452,27 @@ mutation {
...
+
+{{% notice "note:" %}}
+`drop all` operation is executed at cluster level and deletes data, and schema across namespaces. Guardian of the namespace can trigger `drop data` operation within the namespace. The `drop data` operation deletes all the data but retains the schema only. 
 {{% /notice %}}
 
+For example:
+
+```
+curl 'http://localhost:8080/alter' \
+  -H 'X-Dgraph-AccessToken: <your-access -token>' \
```
Comment (danielmai) [inline content/enterprise-features/multitenancy.md:470]:
```suggestion
  -H 'X-Dgraph-AccessToken: <your-access-token>' \
```

File: content/enterprise-features/multitenancy.md [hunk 6]
```diff
@@ -444,16 +452,27 @@ mutation {
...
+The `drop all` operations can be triggered only by a [Guardian of the Galaxy](#guardians-of-the-galaxy).
 They're executed at cluster level and delete data across namespaces.
-All other `drop` operations run at namespace level and are namespace specific.
+All other `drop` operations run at namespace level and are namespace specific. For information about other drop operations, see [Alter the database]({{< relref "raw-http.md#alter-the-database" >}}). 
 
-{{% notice "note" %}}
-`drop all` and `drop data` operations are executed at cluster level and will delete across namespaces. The `drop data` operation will delete all the data but will keep the schema only.
+
+{{% notice "note:" %}}
+`drop all` operation is executed at cluster level and deletes data, and schema across namespaces. Guardian of the namespace can trigger `drop data` operation within the namespace. The `drop data` operation deletes all the data but retains the schema only.
```
Comment (danielmai) [inline content/enterprise-features/multitenancy.md:463]:
```suggestion
`drop all` operation is executed at cluster level and deletes data and schema across namespaces. Guardian of the namespace can trigger the `drop data` operation within the namespace. The `drop data` operation deletes all the data but retains the schema. 
```

**Review by danielmai — APPROVED** [review]
I've added some minor changes. Otherwise, this looks good to me!

File: content/enterprise-features/multitenancy.md (hunk 6 above)
Comment (Rajakavitha1) [reply content/enterprise-features/multitenancy.md:463]: Hi @danielmai I have followed the general and Dgraph Style guide for the usage of comma. "Use commas to separate items in series, and use commas to separate certain kinds of clauses. In a series of three or more items, use a comma before the final and or or."https://discuss.dgraph.io/t/dgraph-developer-documentation-style-guide/10955

Comment (danielmai) [reply content/enterprise-features/multitenancy.md:463]: I didn't think this was a list of three items. The comma here looks like it splits the two item list "data and schema".

Comment (Rajakavitha1) [reply content/enterprise-features/multitenancy.md:463]: Thanks @danielmai !!!

**Pull request merged**